src/test/java/com/epam/tc/hw7/JsonParserFile.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.tc.hw7;
+
+import com.epam.tc.hw7.entities.MetalColorEntity;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class JsonParserFile {
+
+    public LinkedHashMap fileToMap() {
+
+        ObjectMapper objectMapper = new ObjectMapper();
+
+        try (FileReader reader = new FileReader("src/test/resources/JDI_ex8_metalsColorsDataSet.json")) {
+
+            LinkedHashMap<String, MetalColorEntity> map = objectMapper
+                    .readValue(reader, new TypeReference<LinkedHashMap<String, MetalColorEntity>>() {
+                    });
+            return  map;
+
+        } catch (IOException exception) {
+            System.out.println("JSON parsing error: " + exception);
+        }
```

Comment:
It would be better to throw error here
Because we want to know that configuration was not successful and test is broken, instead of `passes`

Reply:
removed try-with-resources construction, added throw exception and closing of FileReader